test/unit/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
why prefer JSON over JS ?

Reply:
This improves compatibility with tools such as VSCode to get better lint feedback within the editor. It seems it can parse and understand the JSON file no problem, but cannot (or will not) execute a JS file.

Reply:
that's wired.. although both json and js are supported, I think js is recommend in general.